### ISSUE TITLE
Unable to migrate data from v.1.0.102

### ISSUE THREAD

**Issue by jacobq11** [Requarks/wiki#1627]
**Describe the bug**
I am having issues migrating my pages from v1.0.102 using the built in utility. It was not installed with git so I need to import from local folder. I was able to confirm the proper directory where the old server is.
 
![wikijs-1](https://user-images.githubusercontent.com/3047639/77694092-b4ddbf00-6f66-11ea-8fd3-3e7b5e3afe9d.PNG)

When I enter the working directory into the import utility, I get the following error:

![wikijs-2](https://user-images.githubusercontent.com/3047639/77694363-1e5dcd80-6f67-11ea-851b-9a3005df1891.PNG)

**To Reproduce**
1. Go to `Administration/Utilities/Import from Wiki.js 1.x`
2. Click on `import from local folder`
3. enter path to old server working dir `/minetwiki`

**Expected behavior**
I expect to see it begin an import process

**Host Info (please complete the following information):**
 **Old instance**
- OS: Ubuntu 16.04
- Wiki.js version: v1.0.102
- Database engine: mariadb

**New instance**
- OS: Ubuntu 16.04
 - Wiki.js version: v2.1.113
 - Database engine: postgre 9.5.19

**Comment by NGPixel** [comment]
You need Node.js 10.12 or later, as specified in the [docs](https://docs.requarks.io/install/requirements)

**Comment by jacobq11** [comment]
I am running the latest version on node.js and I still get the same error.

![wikijs-3](https://user-images.githubusercontent.com/3047639/77807858-5d168500-7046-11ea-95e0-d7ef26d9e495.PNG)

**Comment by NGPixel** [comment]
Your first screenshot says otherwise: 8.16.2

Also the working directory is set as `/minetwiki` and you try to import the folder `/minetwiki`, that doesn't make sense. You should point to the folder containing the markdown files you wish to import from v1.

**Comment by jacobq11** [comment]
So I was still having the issue after updating to 13.12. 

I was able to track it down to the systemctl start script. When I updated Node.js, it installed the new version alongside the old one, also using different file structure.

Whenever I would run wiki.js v2 it would use the old version of node at `/usr/bin/node` instead of the new version at `/usr/local/bin/node`

I just had to change which version of node was used during startup.

PS. Thanks for note able the directory. When I did just `/minetwiki` it took forever importing because it also grabbed a crap load of other files. I had to blow up my db and data folder to redo it with `/minetwiki/repo` to get my original files.
